index.js

```diff
@@ -76,6 +76,13 @@ module.exports = {
     this.jshintrc = app.options.jshintrc;
   },
 
+  postBuild: function () {
```

Comment:
Cool, that is something I have wanted for this exact use-case so happy you are on board. I see the task exists so will look into it.